root/static/scripts/common/components/WorkListEntry.js

```diff
@@ -9,42 +9,61 @@
 
 import * as React from 'react';
 
+import {withCatalystContext} from '../../../../context';
+import RatingStars from '../../../../components/RatingStars';
 import loopParity from '../../../../utility/loopParity';
 
 import ArtistCreditLink from './ArtistCreditLink';
 import ArtistRoles from './ArtistRoles';
 import CodeLink from './CodeLink';
+import DynamicAttributesList from './DynamicAttributesList';
 import EntityLink from './EntityLink';
 
 type WorkListRowProps = {|
-  +hasIswcColumn: boolean,
-  +hasMergeColumn: boolean,
+  +$c: CatalystContextT,
+  +checkboxes?: string,
+  +hasAttributesColumn?: boolean,
+  +hasIswcColumn?: boolean,
+  +hasRatingsColumn?: boolean,
```

Comment:
Replacing `has` with `show` in these three props would also seem more consistent

Reply:
And not sure `Column` is needed since you don't have `showRatingsColumn`?